pyshorteners/shorteners/__init__.py

```diff
@@ -136,6 +135,8 @@ def expand(self, url=None):
     def qrcode(self, width=120, height=120):
         if not self.shorten:
             return None
+        if self.engine == Shorteners.TINYCC:
```

Comment:
you can implement the `qrcode` method inside the `Tinycc` class on `tinycc.py` file. Can you remove this if statement please?